README.md

```diff
@@ -174,9 +174,9 @@ Numeric parameters
   * offset
   * limit
 
-Boolean values
-  * includeSubsections 
-  * homeSectionOnly 
+String values
+  * includeSubsections ("true" | "false") 
```

Comment:
API should accept boolean values and we should cast them before they're used in http_build_query.

So you should detect if the parameters are used, cast them and validate instead of changing docs.